analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ValueTypeShouldImplementIEquatable.cs

```diff
@@ -1,23 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
 
-namespace Tests.Diagnostics
+struct MyStruct // Noncompliant {{Implement 'IEquatable<T>' in value type 'MyStruct'.}}
+//     ^^^^^^^^
 {
-    struct MyStruct // Noncompliant {{Implement 'IEquatable<T>' in value type 'MyStruct'.}}
-//         ^^^^^^^^
-    {
-    }
+}
 
-    struct MyCompliantStruct : IEquatable<MyCompliantStruct> // Compliant
+struct MyCompliantStruct : IEquatable<MyCompliantStruct> // Compliant
```

Comment:
Please add a noncompliant test case where struct implements any other interface with one generic argument, like `IComparable<...>`

Reply:
Done. Both for C# as VB.NET.